website/docs: endpoint devices (#18634)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/endpoint/index.md

---
title: Endpoint stage
---

> This stage integrates with [Endpoint Device](../../../../endpoint-devices/index.mdx) functionality, allowing you to verify whether a device executing a flow is registered with authentik.

Reviewer: ```suggestion
This stage integrates with [Endpoint Device](../../../../endpoint-devices/index.mdx) functionality and allows authentik to verify whether the device executing a flow is registered.
```

> The Endpoint stage fetches [device facts](../../../../endpoint-devices/device-compliance/device-reporting.md#device-facts) via a configured [connector](../../../../endpoint-devices/device-compliance/connectors.md) for use in the flow. These device facts can be used by other stages and policies to make device compliance decisions.

Reviewer: ```suggestion
The Endpoint stage fetches [device facts](../../../../endpoint-devices/device-compliance/device-reporting.md#device-facts) via a configured [connector](../../../../endpoint-devices/device-compliance/connectors.md) and injects them into the flow context. These device facts can be used by other stages and policies to make device compliance decisions.
```

> ### Connector

Reviewer: I mean, this page has no section why not just make it H2s?

> ### Connector

Reviewer: ```suggestion
## Connector
```

> Select the [connector](../../../../endpoint-devices/device-compliance/connectors.md) that the Endpoint stage should use to obtain device information.

Reviewer: ```suggestion
Select the [connector](../../../../endpoint-devices/device-compliance/connectors.md) that the Endpoint stage will use to obtain device facts.
```

> ### Mode

Reviewer: ```suggestion
## Mode
```

> Select whether an endpoint device is required for the stage to succeed or not.

Reviewer: ```suggestion
Select whether the presence of a registered endpoint device is required for the stage to succeed.
```
